test/type-resolution/callables.test.ts

```diff
@@ -29,14 +30,10 @@ test('Static Methods are resolved correctly', async () => {
     },
   });
 
-  const typedTemplate = template
-    .resourceGraph()
-    .map((logicalId, resource) =>
-      resolveResourceLike(resource, logicalId, typeSystem)
-    );
+  const typedTemplate = typed(typeSystem, template);
```

Comment:
Something like this wouldn't need the helper. I forgot that when I added `TypedTemplate`...
(Needs an import)
```suggestion
  const typedTemplate = new TypedTemplate(template, {typeSystem});
```